packages/cli/src/metadataGeneration/transformer/dateTransformer.ts

```diff
@@ -0,0 +1,28 @@
+import * as ts from 'typescript';
+import { Tsoa } from '@tsoa/runtime';
```

Comment:
Note for the future, we should probably remove this namespace and use a module.

Reply:
Once we decouple everything from GIGANTIC type resolver, I think we can do this.